cmd/bun/migrations/20240630181359_add_url_count.up.sql

```diff
@@ -0,0 +1,9 @@
+BEGIN;
+
+--bun:split
+
+ALTER TABLE users ADD  url_count bool  int NOT NULL DEFAULT 0;
```

Comment:
please user upper case for table name, I know above script will work fine but just got good practice

Reply:
done